build.gradle

```diff
@@ -83,6 +89,7 @@ allprojects {
 
     ext {
         hibernateVersion = '5.2.16.Final'
+        elasticSearchVersion = '6.3.0'
     }
```

Comment:
Seemed 6.3.0 are defined in a number of places

Reply:
it's used in the line above, which is the declaration of the variable, and in 
```ext['elasticsearch.version'] = '6.3.0'```
I've tried the following but gradle does not allow that:
```ext['elasticsearch.version'] = "${elasticSearchVersion}"```

I've also tried the following but same I get an error:
```
ext['elasticsearch.version'] = '6.3.0'
ext {
        hibernateVersion = '5.2.16.Final'
        elasticSearchVersion = "${elasticSearchVersion}"
    }
```
Also tried this but it's not working:
```
elasticSearchVersion = '6.3.0'
ext['elasticsearch.version'] = "${elasticSearchVersion}"
ext {
        hibernateVersion = '5.2.16.Final'
        elasticSearchVersion = "${elasticSearchVersion}"
    }
```
I'm giving up :)

Reply:
Thx